auth/provider_config_test.go

```diff
@@ -303,6 +326,15 @@ func TestCreateOIDCProviderConfigInvalidInput(t *testing.T) {
 				ClientID("CLIENT_ID").
 				Issuer("not a url"),
 		},
+		{
+			name: "MissingClientSecret",
+			want: "Client Secret must not be empty for Code Response Type",
```

Comment:
Should we add more test cases for when both are set to `true` and `false`?

Reply:
Done